dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.idl

```diff
@@ -118,6 +118,24 @@ namespace Microsoft.Windows.AppNotifications.Builder
         AppNotificationComboBox SetSelectedItem(String id);
     };
 
+    [contract(AppNotificationBuilderContract, 2)]
+    runtimeclass AppNotificationComboBox2
+    {
```

Comment:
There might be code outthere that relies on the flawed behavior of the initial version of the AppNotificationComboBox builder, hence the need for a new version.

There is also a slight change in the API: while the original AppNotificationComboBox builder was exposing the key value map used to keep the items, this version does not. For one, the new implementation uses two internal data structures: a map to keep the key value pairs and a vector to track the ordering of the items. Both structures must be kept in sync.

There is a point to be made that all the items could be kept in the vector and the map is overkill but doing so would still require some extra code.